docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Containerizing Dell EMC OpenManage Ansible Modules
+Containerized image of Dell EMC OpenManage Ansible Modules can be built using the dockerfile available [here](./Dockerfile). We will be using Docker Engine for containerizing OpenManage Ansible Modules. More information about Docker Engine and install instructions can be found [here](https://docs.docker.com/engine/).
+
+## Build
+Download [Dockerfile](./Dockerfile) to the machine where Docker Engine is installed and run below command. 
+```bash
+docker build -t dellemc/openmanage-ansible-modules .
+```
+Upon success, new image with name _dellemc/openmanage-ansible-modules_ will be created. One can verify successful creation of image using command `docker image ls` which will list newly created image with repository name as _dellemc/openmanage-ansible-modules_.
```

Comment:
Can you add a disclaimer to remind the respective data center administrator to get the required approvals for consuming the docker images and other packages from open source repositories?

Reply:
We are shipping any docker image. Consumers of this file need to be aware of inherent security risks associated with docker images and containers. Will add relevant link to readme.

Reply:
@saishivarcr; Better we should have the over all disclaimer stating that the respective open source component used in your script, before building the image out of it, get the required approvals from your management. Dell as such won't be responsible for in case anything. This is because the script is pulling out various open source bundles such as docker base image of python:slim3 and ansible which are not dellemc deliverables.

Reply:
readme updated with this information